docs/technical_reference/property_models/seawater_ion_generic.rst

```diff
@@ -0,0 +1,87 @@
+Seawater Ion Generic Property Package
+=====================================
+
+This package implements property relationships for artificial seawater as provided in `Islam et al. (2021) <https://www.sciencedirect.com/science/article/abs/pii/S1383586621009412>`_.
```

Comment:
The main purpose of this model was to make use of IDAES' eNRTL model, which should be emphasized here. We then used the eNRTL model to predict mineral scaling in full treatment where RO was the primary desalination technology.

Another detail to mention: the formulation of this model is based what was formerly referred to as IDAES' generic property framework, which I think has now been renamed to something like the modular property framework.

Reply:
More specifically, `seawater_ion_generic` is really just a configuration file that gets passed into the GenericParameterBlock (generic property framework I mentioned earlier). You can look at the test for `seawater_ion_generic` to see what I mean. So perhaps clarification should be provided here to explain this. I am also wondering if `seawater_ion_generic` should be in a different folder at this point, but I guess that's a different discussion.